src/containers/Tenant/Query/Preview/Preview.scss

```diff
@@ -1,10 +1,9 @@
-@use '../../../../styles/mixins.scss';
+@import '../../../../styles/mixins.scss';
 
 .kv-preview {
     height: 100%;
-    @include mixins.flex-container();
-    @include mixins.query-data-table();
-
+    @include flex-container();
+    @include query-data-table();
     &__header {
```

Comment:
Please, add property `flex-shrink: 0;` to header, now it looks different, depending if there is result or not

Reply:
With result:
![Screenshot 2024-11-27 at 17 01 48](https://github.com/user-attachments/assets/4404dba0-4594-40f4-92b0-7fccec780c64)

Without result:
![Screenshot 2024-11-27 at 17 01 59](https://github.com/user-attachments/assets/39605021-6a72-4b37-9e03-9319cd4f469d)